docs/UserGuide.md

```diff
@@ -287,10 +310,16 @@ Format: `delete INDEX`
 * Deletes the applicant at the specified `INDEX`. The index refers to the index number shown in the displayed applicant list.
 * `INDEX` must be a positive integer (e.g. `1`, `2`, `3`, ...).
 
-Example:
+Valid examples:
 * `list` followed by `delete 2` deletes the 2nd person in the applicant list.
 * `sort d/name` followed by `delete 3` deletes the 3rd person in the sorted applicant list.
 
+Invalid examples:
+* `delete`
+This is not allowed as an `INDEX` must be specified.
+* `delete -3`
+This is not allowed as an `INDEX` must be a positive integer.
```

Comment:
i think this could be changed to not allowed as **the** index must be ... to be consistent with the rest of the ug